### ISSUE TITLE
Use new GSV API in core Sidewalk?

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2891]
Label Gallery uses the newer GSV API that has nicer UI controls, imo, for navigation:
![image](https://user-images.githubusercontent.com/1621749/167957292-59d0ec8b-a1cc-4fbc-8689-a6c14a617be1.png)

Core Sidewalk, however, still uses the old GSV API, which has those solid white arrows for navigation:
![image](https://user-images.githubusercontent.com/1621749/167957406-0a7ab5d1-a948-4434-a8bc-ba95b7716fd5.png)

@misaugstad and I discussed this in our 1:1 and he remembered previously exploring the newer GSV API but can't remember why it was not integrated into core.

**Comment by jonfroehlich** [comment]
Couldn't we artificially limit this as just taking a "step" in the next direction if the initial "step" looks too long?

**Comment by misaugstad** [comment]
I don't think that we have the ability to intercept the pano change before GSV does it. I think that the best we could do is let the pano load that they tried to go to, and then load a new panorama that is actually only one step forward.

**Comment by jonfroehlich** [comment]
Alternative idea: maybe we could also pop up a notification if they take too long of a step?

**Comment by jonfroehlich** [comment]
I'd like to consider this in the future as I do think the new GSV controls are much more intuitive to use.

**Comment by jonfroehlich** [comment]
Note: In my work at Google with the GSV API, I know this is possible. So, we should reconsider this Issue.
